certora/applyMunging.patch

```diff
@@ -114,7 +114,20 @@ diff -ruN MetaMorpho.sol MetaMorpho.sol
  
      /// @notice OpenZeppelin decimals offset used by the ERC4626 implementation.
      /// @dev Calculated to be max(0, 18 - underlyingDecimals) at construction, so the initial conversion rate maximizes
-@@ -126,7 +122,7 @@
+@@ -107,6 +103,12 @@
+     /// @inheritdoc IMetaMorphoBase
+     uint256 public lastTotalAssets;
+ 
++    // HARNESS
++    mapping(Id => uint256) public withdrawRank;
++
++    // HARNESS
++    mapping(Id => uint256) public deletedAt;
```

Comment:
let's add little "natspec" here, as their exact value isn't so clear from the name